### ISSUE TITLE
Merging the new manual with the new ocamlbuild repo

### ISSUE THREAD

**Issue by rleonid** [gasche/manual-ocamlbuild#27]
A question and not an issue, but are there plans/timeline to merge this manual with the newly created separate ocamlbuild repo? You've made tremendous progress on this manual (Thank you!) and I think trying to keep the two projects in sync would be valuable, though I can imagine arguments for keeping them separate.

**Comment by gasche** [comment]
Yes, there is a plan: I personally had no strong opinion one way or another, but I observed that the same move for the OCaml distribution ( ocaml/ocaml#302 ) has been very successful in motivating people to edit the documentation. And it certainly makes sense to have both a feature and its documentation in the same commit.

On the other hand, I', not sure the draft of new manual in it's current state is ready for a release. Any help to polish it, or feedback of the form "I had a look and I think X and Y need to be done to get it in release-ready state", would be appreciated.

My priority right now is to have an experimental release real soon ( #32 ), so I won't work on the manual in the following weeks. If someone is interested in sending a PR to do the merge, they are warmly welcome.

**Comment by rleonid** [comment]
Cool, after #32 then.

**Comment by gasche** [comment]
You should keep the issue open, it's useful to track the task, and I use labelling to know what is urgent so it does not add much overhead.

**Comment by rleonid** [comment]
No problem.
